addon/components/o-s-s/empty-state.ts

```diff
@@ -0,0 +1,19 @@
+import type { htmlSafe } from '@ember/template';
+import Component from '@glimmer/component';
+
+interface OSSEmptyStateComponentSignature {
+  image?: string;
+  title?: ReturnType<typeof htmlSafe>;
+  subtitle?: ReturnType<typeof htmlSafe>;
+  size?: 'sm' | 'md';
+}
+
+export default class OSSEmptyStateComponent extends Component<OSSEmptyStateComponentSignature> {
+  get titleSize(): string {
+    return this.args.size === 'sm' ? 'md' : 'lg';
+  }
+
+  get subtitleSize(): string {
+    return this.args.size || 'md';
+  }
+}
```

Comment:
you could add a `size` getter that returns `this.args.size ?? 'md'` and use it locally. or even:

```
get size(): string {
  return ['sm', 'md'].includes(this.args.size) ? this.args.size : 'md';
}
```

(so we properly handle invalid sizes)